lib/tasks_private/vpc.rake

```diff
@@ -0,0 +1,168 @@
+namespace :spec do
+  require 'ibm_cloud_sdk_core'
+  require 'ibm_cloud_resource_controller'
+  require 'ibm_vpc'
+
+  base_dir = File.join(ManageIQ::Providers::IbmCloud::Engine.config.root.to_s, 'spec')
```

Comment:
You can leave this as a Pathname which makes it a little cleaner to build the paths later:

```
base_dir = ManageIQ::Providers::IbmCloud::Engine.root.join("spec")
```